src/shell_executor.rs

```diff
@@ -39,11 +40,12 @@ impl CommandExecutor {
         self.execute(
             "sudo",
             &[
-                "ip",
-                "link",
-                "set",
-                interface,
-                if enable { "up" } else { "down" },
+                "echo"
```

Comment:
The only reason I’m disabling interfaces is because I don’t want the scoreboard to constantly be broadcasting a WiFi connection. Does this still do that?

Reply:
It doesn't. I'm working on a better way to disable the WIFIs